README.md

```diff
@@ -32,11 +32,51 @@ iris.each do |record|
      record.petal_width,
      record.class,
   ]
-  # [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
-  # [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 end
+# => [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
+# => [4.9, 3.0, 1.4, 0.2, "Iris-setosa"]
+  :
+# => [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 ```
 
+Here is an example to access iris dataset by `Table#to_h`:
+
+```ruby
+require "datasets"
+
+iris = Datasets::Iris.new
+iris_table = iris.to_table.to_h
+p iris_table[:sepal_length]
+# => [5.1, 4.9, .. , 7.0, ..
+p iris_table[:sepal_width]
+# => [3.5, 3.0, .. , 3.2, ..
+p iris_table[:petal_length]
+# => [1.4, 1.4, .. , 4.7, ..
+p iris_table[:petal_width]
+# => [0.2, 0.2, .. , 1.4, ..
+
+p iris_table[:class]
+# => ["Iris-setosa", "Iris-setosa", .. , "Iris-versicolor", ..
+```
+
+Here is an example to access iris dataset by `Table#fetch_values`:
+
+```ruby
+require "datasets"
+
+iris = Datasets::Iris.new
+p iris.to_table.fetch_values(:sepal_length, :sepal_width, :petal_length, :petal_width).transpose
```

Comment:
How about defining `iris_table = iris.to_table` local variable?
Calling `to_table` for each `fetch_values` is costful.

Reply:
Thank you very much.
Fixed.